examples/mongodbatlas_resource_policy/README.md

```diff
@@ -0,0 +1,12 @@
+# MongoDB Atlas Provider -- Atlas Resource Policy
+This example creates three different resource policies in an organization.
+
+**NOTE**: Resource Policies are currently in Public Preview. You will need to
```

Comment:
same comment as on first file

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2